pkg/api/types_test.go

```diff
@@ -6571,97 +6555,7 @@ func TestAnyAgentUsesAvailabilitySets(t *testing.T) {
 		test := test
 		t.Run(test.name, func(t *testing.T) {
 			t.Parallel()
-			ret := test.p.AnyAgentUsesAvailabilitySets()
-			if test.expected != ret {
-				t.Errorf("expected %t, instead got : %t", test.expected, ret)
-			}
-		})
-	}
-}
-
-func TestAnyAgentUsesVirtualMachineScaleSets(t *testing.T) {
-	tests := []struct {
-		name     string
-		p        *Properties
-		expected bool
-	}{
-		{
-			name: "one agent pool w/ AvailabilitySet",
-			p: &Properties{
-				AgentPoolProfiles: []*AgentPoolProfile{
-					{
-						Name:                "agentpool1",
-						VMSize:              "Standard_D2_v2",
-						Count:               2,
-						AvailabilityProfile: AvailabilitySet,
-					},
-				},
-			},
-			expected: false,
-		},
-		{
-			name: "two agent pools, one w/ AvailabilitySet",
-			p: &Properties{
-				AgentPoolProfiles: []*AgentPoolProfile{
-					{
-						Name:                "agentpool1",
-						VMSize:              "Standard_D2_v2",
-						Count:               2,
-						AvailabilityProfile: AvailabilitySet,
-					},
-					{
-						Name:   "agentpool1",
-						VMSize: "Standard_D2_v2",
-						Count:  100,
-					},
-				},
-			},
-			expected: false,
-		},
-		{
-			name: "two agent pools",
-			p: &Properties{
-				AgentPoolProfiles: []*AgentPoolProfile{
-					{
-						Name:   "agentpool1",
-						VMSize: "Standard_D2_v2",
-						Count:  2,
-					},
-					{
-						Name:   "agentpool1",
-						VMSize: "Standard_D2_v2",
-						Count:  100,
-					},
-				},
-			},
-			expected: false,
-		},
-		{
-			name: "two agent pools, one w/ VirtualMachineScaleSets",
-			p: &Properties{
-				AgentPoolProfiles: []*AgentPoolProfile{
-					{
-						Name:   "agentpool1",
-						VMSize: "Standard_D2_v2",
-						Count:  2,
-					},
-					{
-						Name:                "agentpool1",
-						VMSize:              "Standard_D2_v2",
-						Count:               100,
-						AvailabilityProfile: VirtualMachineScaleSets,
-					},
-				},
-			},
-			expected: true,
-		},
-	}
-
-	for _, test := range tests {
-		test := test
-		t.Run(test.name, func(t *testing.T) {
-			t.Parallel()
-			ret := test.p.AnyAgentUsesVirtualMachineScaleSets()
+			ret := test.p.HasVMASAgentPool()
```

Comment:
Wrong substitution?

Reply:
no it's just that I removed the whole test for the function I deleted so the diff appears weird. Test for AnyAgentUsesVirtualMachineScaleSets is completely gone, test for HasVMASAgentPool starts on line 6476 and continues here.

Reply:
or maybe the diff is confusing here